src/qt/walletview.cpp

```diff
@@ -20,6 +20,7 @@
 #include <qt/walletmodel.h>
 
 #include <interfaces/node.h>
+#include <psbt.h>
```

Comment:
Might need to drop 11... Weird that it didn't do the other part of the commit, yet merged smoothly

Reply:
agree, let's drop it for now